addons/RAWR/RAWR.lua

```diff
@@ -0,0 +1,43 @@
+_addon.name = 'RAWR'
+_addon.author = 'Genoxd'
+_addon.version = '1.0.0.0'
+_addon.commands = {'rawr'}
+
+require 'lists'
```

Comment:
You should change this to `require('tables')`, since you don't actually use any lists in here, only the Windower-specific tables.